radio/src/telemetry/frsky_pxx2.cpp

```diff
@@ -227,6 +227,11 @@ void processAuthenticationFrame(uint8_t module, uint8_t * frame)
   uint8_t cryptoType = frame[3];
   uint8_t messageDigest[16] = {0};
 
+  if (0x04 == frame[0] && 0xA5 == frame[4]) {
```

Comment:
Could you please add “defines” for your constants?

Reply:
Which file should I put the 'defines' in ? How about the strings ?

Reply:
`pulses/pxx2.h` would be a good place, next to the other frame defs. Thx!

Reply:
I will take it

Reply:
Now I read it more carefully, the 0x04 is not a flag, it's just a sanity check, I have changed it to `frame[0] >= 4`